claim-check/code-samples/sample-2/azure-function/ClaimCheck.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Microsoft.Azure.EventGrid;
+using Microsoft.Azure.EventGrid.Models;
+
+namespace Microsoft.PnP.Messaging
+{
+    public static class ClaimCheck
+    {    
+        [FunctionName("ClaimCheck")]
```

Comment:
Make this Azure Function look like Sample 1

Reply:
Done